utils/chain-spec-builder/src/main.rs

```diff
@@ -258,9 +287,11 @@ fn genesis_constructor(
         versioned_store_permissions_cfg,
         data_directory_config,
         content_working_group_config,
+        initial_account_balances,
     )
 }
 
```

Comment:
Please add a small comment on why we disable the lint here.

Reply:
done